pandas/core/common.py

```diff
@@ -37,7 +37,7 @@
 )
 from pandas.core.dtypes.generic import ABCExtensionArray, ABCIndex, ABCSeries
 from pandas.core.dtypes.inference import iterable_not_string
-from pandas.core.dtypes.missing import isna, isnull, notnull  # noqa
+from pandas.core.dtypes.missing import isna, isnull, notnull  # noqa:F401
```

Comment:
we should remove these imports entirely if they are not being used (this was a holdover from a long time ago i think)

Reply:
sure, will add that to the next assorted-cleanups branch ive got going

Reply:
ended up adding that here